ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml

```diff
@@ -131,6 +139,40 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: generic-claim
+- as: fips-image-scan-operator
+  optional: true
+  steps:
+    dependencies:
+      SCAN_IMAGE: cincinnati-operator
+    test:
+    - ref: fips-check-image-scan
+- as: fips-image-scan-operator-production
+  cron: '@weekly'
+  steps:
+    test:
+    - as: check-payload
+      commands: |
+        set -o nounset
+        set -o errexit
+        set -o pipefail
+
+        set -x
+        SCAN_IMAGE="registry.redhat.io/openshift-update-service/openshift-update-service-rhel8-operator:latest"
```

Comment:
Ideally, instead of weekly, this job should be triggered whenever there is a new build for `registry.redhat.io/openshift-update-service/openshift-update-service-rhel8-operator:latest`.
But we do not have such a mechanism for CI.

I can work on [fips-check-image-scan](https://steps.ci.openshift.org/reference/fips-check-image-scan) to support external image and custom `$REGISTRY_AUTH_FILE`.